README.md

```diff
@@ -80,6 +80,20 @@ If you are updating, run the same uninstall script, but keep all configuration f
 
 We are adding support for [Orange Pi](http://www.orangepi.org/) boards and have tested with the [Orange Pi One](https://dl.armbian.com/orangepione/nightly/) and [Orange Pi Lite](https://dl.armbian.com/orangepilite/nightly/). Instead of flashing Raspbian, start with the Armbian nightly images linked above, then follow the same installation steps as the Raspberry Pi.
 
+### Hardware Table
+
+Following is a list of hardware tested with the install.
+
+| Hardware                  | Base OS         | CJDNS Bench   | USB  | Ethernet| Notes            |
+| :-------------------------|:----------------|:--------------|:-----|:--------|:-----------------|
+| Raspbery Pi A+            | Rasbian Lite    |               | 1    | None    |                  |
```

Comment:
`s/Raspbery/Raspberry` `s/Raspbian/Raspbian`
same with next two lines. I know you are an Orange guy but wth :P

Reply:
Copy And Paste and i think you mean s/Rasbian/Raspbian/

Corrected error and replaced with upgraded chart.